nixos/modules/services/networking/thelounge.nix

```diff
@@ -74,6 +88,20 @@ in
         <literal>pkgs.theLoungePlugins.plugins</literal> and <literal>pkgs.theLoungePlugins.themes</literal>.
       '';
     };
+
+    ensureUsers = mkOption {
+      default = { };
+      type = types.attrsOf (types.submodule userOptions);
+      example = literalExpression ''
+        john = {
+          password = "$2b$05$fK.qf8vOxvOeSA4D3W0znOhkASuXTceypm6uwqPkrAHh7bWaji7U2";
+        };
+      '';
+      description = ''
+        Ensures that the specified users exist. If one does not, a user with the specified options will be created.
+        This option will not delete or update existing users.
+      '';
```

Comment:
I'm not sure how much I like this wording; I think it can be improved, but I don't know how.